spec/system/grant_reviewers/grant_reviewer_invitations_spec.rb

```diff
@@ -1,9 +1,125 @@
 require 'rails_helper'
 
 RSpec.describe GrantReviewer::Invitation, type: :system do
-  context 'paper trail' do
-    pending 'it tracks whodunnit' do
```

Comment:
Did we want to keep this pending spec?

Reply:
yeah, i'm going to leave it as a reminder that it should be fixed. could see more specs for this down the line (e.g. reminders). there are a fair number of pending specs -- including empty spec files -- that should be reviewed at some point.